### PR TITLE
Modiffied the tests for manual data and metadata aquisition

### PR DESCRIPTION
**Pull request by PalomoIFCA**
Added config.ini links and corrected the tests

### CODE REVIEW

File: plugins/epos/plugin.py [hunk 1]
```diff
@@ -265,18 +265,40 @@ def rda_a1_02m(self):
...
+        -------
+        points
+            A number between 0 and 100 to indicate how well this indicator is supported
+        msg
+            Message with the results or recommendations to improve this indicator
+        """
+        # 2 - Look for the metadata terms in HTML in order to know if they can be accessed manually
+
+        msg = "The link to the manual obtention  of the data is " + str(
+            self.manual_data
```
Comment (orviz) [inline plugins/epos/plugin.py:299]: where is this variable defined?

File: plugins/epos/plugin.py [hunk 2]
```diff
@@ -265,18 +265,40 @@ def rda_a1_02m(self):
         """
         # 2 - Look for the metadata terms in HTML in order to know if they can be accessed manually
 
-        return (0, "")
+        msg = "The link to the manual obtention  of the metadata is " + str(
+            self.manual_metadata
```
Comment (orviz) [inline plugins/epos/plugin.py:269]: where is this variable defined?

**Review by orviz — CHANGES_REQUESTED** [review]

File: plugins/epos/plugin.py (hunk 1 above)
Comment (PalomoIFCA) [reply plugins/epos/plugin.py:301]: It dissapeared while separating the PRs

File: plugins/epos/plugin.py (hunk 2 above)
Comment (PalomoIFCA) [reply plugins/epos/plugin.py:271]: It dissapeared while separating the PRs. Now it is definned after getting the metadata

**Review by orviz — CHANGES_REQUESTED** [review]

File: plugins/epos/plugin.py [hunk 3]
```diff
@@ -263,20 +269,40 @@ def rda_a1_02m(self):
         msg
             Message with the results or recommendations to improve this indicator
         """
-        # 2 - Look for the metadata terms in HTML in order to know if they can be accessed manually
 
-        return (0, "")
+        msg = "The link to the manual obtention  of the metadata is " + str(
```
Comment (orviz) [inline plugins/epos/plugin.py:273]: Need to check first that `self.metadata_access_manual` is not empty before assigning the check 100 points

**Review by orviz — CHANGES_REQUESTED** [review]

File: plugins/epos/plugin.py [hunk 4]
```diff
@@ -260,19 +266,46 @@ def rda_a1_02m(self):
         msg
             Message with the results or recommendations to improve this indicator
         """
-        # 2 - Look for the metadata terms in HTML in order to know if they can be accessed manually
 
-        return (0, "")
+        points = 0
+        msg = "No link to the manual obtention of the metadata"
+        if not self.metadata_access_manual == "":
```
Comment (orviz) [inline plugins/epos/plugin.py:272]: better `if self.metadata_access_manual`

File: plugins/epos/plugin.py [hunk 5]
```diff
@@ -260,19 +266,46 @@ def rda_a1_02m(self):
...
+        Returns
+        -------
+        points
+            A number between 0 and 100 to indicate how well this indicator is supported
+        msg
+            Message with the results or recommendations to improve this indicator
+        """
+        points = 0
+        msg = "No link to the manual obtention of the data"
+        if not self.metadata_access_manual == "":
```
Comment (orviz) [inline plugins/epos/plugin.py:304]: same comment as above

**Review by orviz — CHANGES_REQUESTED** [review]

**Review by orviz — APPROVED** [review]

**Pull request merged**